### DISCUSSION TITLE
Should chat history allow me to pick up conversations where I left off?

### DISCUSSION THREAD

**Discussion by upnix** [brianpetro/obsidian-smart-connections#704] [Q&A]
In the chat pane, I click the "Chat History" button, pick a saved chat, and nothing happens.

I thought I remembered this re-populating the chat history with the contents of that chat. In the least, however, I'd expect _something_ to happen when I select a saved chat. I've tried this in a second vault and get the same result.

Can someone set me straight on this? What happens for you when you select a saved chat using the "Chat History" button?

**Comment by upnix** [comment]
I'll leave this here, but I discovered where I went wrong. I changed the path where saved chats were stored in Smart Connections, so it was still showing past chats in my history, but it was unable to load them.

Not sure how to fix that for past chats, as I just deleted them all and started over with my new saved chat path in place.

**Reply by brianpetro** [reply]
Hey @upnix 

Thanks for the update. I was just looking into this.

Do you know if you restarted after it stopped working as expected? Also, did you use the settings to change the folder name, or did you change it directly through the Obsidian file explorer?

Thanks for bringing this to my attention and the follow-up
🌴

**Reply by upnix** [reply]
I lived with it working like this for a while, so I would have restarted lots of times. I used the settings to change the folder path, but followed up by moving the old folders to their new location.

The way I discovered the problem was that after trying to load an old chat, I tried to then use the chat and got this error message:
![image](https://github.com/user-attachments/assets/4888b2c9-96fa-4326-a404-9b096016ad6a)

And realized that it was using the old path of the chat, but then adding it to the new path.

To add on to this - this is my current configuration for this path:
![image](https://github.com/user-attachments/assets/150cff2a-e7e0-41e7-8df6-fc02ee334bb2)

I think by default `chats` is placed in the Obsidian root folder.

**Reply by brianpetro** [reply]
@upnix gotcha. And thanks for the additional details, I'll use them to do some testing to figure out what might not be working as expected.

🌴

**Comment by upnix** [comment]
An update to this - the problem was not simply an issue of old chats created under a different configuration. After I restart Obsidian all previously saved chats will show up when I click the "History" icon with a path of `/Smart Connections/chats/...`, none of which will load when I click on them. New chat histories I save are not prefixed with that path and will load, that is, until I restart Obsidian.

![image](https://github.com/user-attachments/assets/8449215f-c857-4ed2-a056-9b0917059d5b)

**Reply by brianpetro** [reply]
@upnix, thanks for the update on this bug. I'm going to have to dig into it a little further, but it's safe to say that it needs to be fixed. I created an issue to track this at https://github.com/brianpetro/obsidian-smart-connections/issues/706
